Features/HelloRequest.cs

```diff
@@ -0,0 +1,11 @@
+using MediatR;
+using FluentResults;
+
+namespace HelloApi.Features
+{
+	public class HelloRequest : IRequest<Result<string>>
+	{
+		// Inicializa a propriedade para evitar warning de valor nulo
+		public string Name { get; set; } = string.Empty;
+	}
```

Comment:
record -> representa uma estrutura imutável, suas propriedades não mudam
classe -> representa uma estrutura mutável, onde suas propriedades podem ser alteradas

sealed -> diz que uma estrutura final, que não pode ser herdada
abstract -> diz que uma estrutura não pode ser instanciada, apenas herdada

set: permite que a propriedade seja alterada/setada
init: a propriedade somente é definida no momento da criação do objeto

```suggestion
	public sealed record HelloRequest(string Name) : IRequest<Result<string>>
```